common/src/main/scala/uk/ac/wellcome/models/MergedSierraRecord.scala

```diff
@@ -38,8 +42,23 @@ case class MergedSierraRecord(
       None
     }
   }
+
+  override def transform: Try[Work] =
+    JsonUtil.fromJson[SierraBibData](bibData.get.data).map { sierraBibData =>
```

Comment:
Should we have a branch that recovers in the case that `bibData.get` is None? And then test for that?